account-kit/core/src/utils/cookies.ts

```diff
@@ -9,11 +9,12 @@ import { deserialize } from "./deserialize.js";
  * Function to create cookie based Storage
  *
  * @param config optional config object that allows you to set the session length
+ * @param config.sessionLength the length of the session in milliseconds
  * @returns an instance of a browser storage object that leverages cookies
  */
-export const cookieStorage: (config?: { sessionLength: number }) => Storage = (
```

Comment:
I need to handle this case too because it's represented differently in the AST. Doing it this way causes some issues with generating the docs and I'll need to fix it